entity-framework/core/querying/related-data.md

```diff
@@ -109,6 +109,27 @@ Contents of `School` navigation of all People who are Students can be eagerly lo
   context.People.Include("School").ToList()
   ```
 
+### Filtered include
```

Comment:
We should consider putting this just below "Including multiple levels", above "Include on derived types". It connects more naturally to including multiple levels and seems generally more relevant to people.